pkg/services/sqlstore/migrator/dialect.go

```diff
@@ -49,6 +49,9 @@ type Dialect interface {
 
 	IsUniqueConstraintViolation(err error) bool
 	IsDeadlock(err error) bool
+
+	// modifier is only useful for sqlite
+	ToTimestamp(d1 string, modifier string) string
```

Comment:
Given my suggestion of storing and comparing epochs instead of DateTime this can be removed.